packages/twenty-front/src/modules/ui/input/components/internal/phone/components/PhoneCountryPickerDropdownButton.tsx

```diff
@@ -95,7 +103,9 @@ export const PhoneCountryPickerDropdownButton = ({
         <StyledDropdownButtonContainer isUnfolded={isDropdownOpen}>
           <StyledIconContainer>
             {selectedCountry ? <selectedCountry.Flag /> : <IconWorld />}
-            <IconChevronDown size={theme.icon.size.sm} />
+            <StyledCheveronIconContainer>
```

Comment:
Why did you add a styled container?

Reply:
The chevron down icon is not sized as per the figma file, the only way to change the size is by adding a container